packages/core/core-flows/src/order/workflows/return/confirm-return-request.ts

```diff
@@ -36,6 +47,69 @@ const validationStep = createStep(
   }
 )
 
+function prepareFulfillmentData({
+  order,
+  items,
+  returnShippingOption,
+}: {
+  order: OrderDTO
+  items: any[]
+  returnShippingOption: {
+    id: string
+    provider_id: string
+    service_zone: {
+      fulfillment_set: {
+        location?: {
+          id: string
+          address: Record<string, any>
+        }
+      }
+    }
+  }
+}) {
+  const orderItemsMap = new Map<string, Required<OrderDTO>["items"][0]>(
+    order.items!.map((i) => [i.id, i])
+  )
+  const fulfillmentItems = items.map((i) => {
+    const orderItem = orderItemsMap.get(i.item_id)!
+    return {
+      line_item_id: i.item_id,
+      quantity: i.quantity,
+      return_quantity: i.quantity,
+      title: orderItem.variant_title ?? orderItem.title,
+      sku: orderItem.variant_sku || "",
+      barcode: orderItem.variant_barcode || "",
+    } as FulfillmentWorkflow.CreateFulfillmentItemWorkflowDTO
+  })
+
+  const locationId =
```

Comment:
With #8302 merged, we can rely on `return.location_id` here

Reply:
I will be able to remove a request after the changes are applied, but here I can keep like this as we are also using the address